containers/podman/down.sh

```
@@ -50,4 +50,40 @@ if [[ "${1:-}" == "--wipe" ]]; then
+            print outside_ns + (cuid - inside_ns)
+            found = 1
+            exit
+          }
+        }
+        END { exit !found }
+      ' /proc/self/uid_map) || return 1
+      echo "$mapped"
+    }
+    mapped=$(_host_uid_for_container_uid 1001 2>/dev/null) || mapped=""
+    if [[ -n "$mapped" ]]; then
+      local state_file="${APME_CACHE_HOST_PATH:-${XDG_CACHE_HOME:-$HOME/.cache}/apme}/.traversal-acls"
```

Shellcheck (0.11.0)

[error] 'local' is only valid in functions.

(SC2168)